IDE/STM32Cube/wolfssl_example.c

```diff
@@ -105,10 +105,9 @@
 #endif /* WOLFSSL_STATIC_MEMORY */
 
 
-/* UART definitions */
-#ifndef HAL_CONSOLE_UART
-#define HAL_CONSOLE_UART huart4
-#endif
+/* This sets which UART to use for the console.  It is something you will have
+ * to configure in STMCubeIDE and then change here. */
+#define HAL_CONSOLE_UART huart3
```

Comment:
Please add back the `#ifndef HAL_CONSOLE_UART`

Reply:
Fixed.